api/v1alpha1/eraserconfig_types.go

```diff
@@ -143,14 +148,30 @@ type Components struct {
 // EraserConfig is the Schema for the eraserconfigs API.
 type EraserConfig struct {
 	metav1.TypeMeta `json:",inline"`
-
-	// ControllerManagerConfigurationSpec returns the configurations for controllers
-	cfg.ControllerManagerConfigurationSpec `json:",inline"`
-
-	Manager    ManagerConfig `json:"manager"`
-	Components Components    `json:"components"`
+	Manager         ManagerConfig `json:"manager"`
+	Components      Components    `json:"components"`
 }
 
 func init() {
 	SchemeBuilder.Register(&EraserConfig{})
 }
+
```

Comment:
can we add a brief comment on why we adding manual conversion

Reply:
done